src/mixins/state.js

```diff
@@ -26,14 +26,14 @@ export default {
    */
   initState(options = {}) {
```

Comment:
I know this isn't exactly related to this PR, but `initState` should also `return this;` for consistency.  Not that _we_ want to, but then  `restart` could be `this.start(options).initState(options).stop(options);` which is something some people will enjoy.